scss/atoms/typography/_typography.scss

```diff
@@ -12,10 +12,10 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( max-width: $viewport-sm ) {
 		font-size: 16px;
 	}
-	@media ( max-width: 500px ) {
+	@media ( max-width: $viewport-xs ) {
```

Comment:
@cvrebert yes it should be min, but it's kinda desktop first at the moment, with the default size applicable for anything larger than 720px screen width. We just decided in a meeting to go mobile first, I just thought that should be separate PR. But since both of them would be so tiny I should probably just push to this.